participants/pfieffer/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Web Storage Example</title>
+	<link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+    <h1>Web Storage Example</h1>
+    <h2>I can remember what you type below, even if you close the browser &#128521</h1>
+
+    <p>Enter a value:</p>
+    <input type="text" id="inputValue">
+    <button onclick="saveToLocalStorage()">Save</button>
+
+    <p>Saved Value:</p>
+    <div id="savedValue"></div>
```

Comment:
You have added Saved Value text 2 times, one in JS and one in HTML. You can remove it from any one place.

Reply:
Hi. Thank you for pointing out, I will get rid of the html paragraph